### PR TITLE
feat: Implements send-side bandwidth estimations.

### PR DESCRIPTION
**Pull request by gpolitis**
Depends on https://github.com/jitsi/libjitsi/pull/361

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/RtpChannel.java [hunk 1]
```diff
@@ -1412,6 +1412,8 @@ public void setRtpHeaderExtensions(
     {
         if ((rtpHeaderExtensions != null) && (rtpHeaderExtensions.size() > 0))
         {
+            stream.clearRTPExtensions();
```
Comment (bbaldino) [inline src/main/java/org/jitsi/videobridge/RtpChannel.java]: interesting, so not having this before was a bug, ya?

File: src/main/java/org/jitsi/videobridge/VideoChannel.java [hunk 2]
```diff
@@ -336,6 +336,8 @@ void initialize(RTPLevelRelayType rtpLevelRelayType)
     {
         super.initialize(rtpLevelRelayType);
 
+        bitrateController.update(null, -1);
```
Comment (bbaldino) [inline src/main/java/org/jitsi/videobridge/VideoChannel.java]: so this obsolesces https://github.com/jitsi/jitsi-videobridge/pull/527, ya?

Comment (gpolitis) [reply src/main/java/org/jitsi/videobridge/VideoChannel.java]: right, just closed it.

File: src/main/java/org/jitsi/videobridge/RtpChannel.java (hunk 1 above)
Comment (gpolitis) [reply src/main/java/org/jitsi/videobridge/RtpChannel.java]: Not a bug, it was just not being used.

**Pull request merged**